src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task {
```

Comment:
Overall coding quality is good. Naming conventions are followed and methods are described with action words accurately, making it easy to understand what they do